packages/lambdas/src/ihe-gateway-v2-inbound-document-query.ts

```diff
@@ -21,7 +21,10 @@ export async function handler(event: APIGatewayProxyEventV2) {
   try {
     if (!event.body) return buildResponse(400, { message: "The request body is empty" });
     try {
-      const dqRequest: InboundDocumentQueryReq = await processInboundDqRequest(event.body);
+      const body = event.isBase64Encoded
+        ? Buffer.from(event.body, "base64").toString()
```

Comment:
nit: in other places in the code base we either use:
`Buffer.from(doc).toString("base64");`
or
`Buffer.from(note, "base64").toString("utf-8");`

Reply:
ya i assumed so